pkg/connector/database_users.go

```diff
@@ -240,31 +282,37 @@ func (o *databaseUserBuilder) CreateAccountCapabilityDetails(ctx context.Context
 	}, nil, nil
 }
 
-func (o *databaseUserBuilder) createUserIfNotExists(ctx context.Context, orgId, email string, profile map[string]any) error {
+func (o *databaseUserBuilder) createUserIfNotExists(ctx context.Context, orgId, email string, profile map[string]any) (string, error) {
 	l := ctxzap.Extract(ctx)
 
-	_, httpResponse, err := o.client.OrganizationsApi.CreateOrganizationInvitation(
+	userInvite, httpResponse, err := o.client.MongoDBCloudUsersApi.CreateOrganizationUser(
```

Comment:
nit: is this the invite or organization user?

Reply:
Org User